Source/Menu/MainForm.Designer.cs

```diff
@@ -84,6 +84,8 @@ private void InitializeComponent()
             this.comboBoxDifficulty = new System.Windows.Forms.ComboBox();
             this.label8 = new System.Windows.Forms.Label();
             this.panelModeTimetable = new System.Windows.Forms.Panel();
+            this.lableTimetableWeatherFile = new System.Windows.Forms.Label();
```

Comment:
```suggestion
            this.labelTimetableWeatherFile = new System.Windows.Forms.Label();
```
(Typo in name of the label)